test/storage/sql_table_test.cpp

```diff
@@ -38,62 +39,134 @@ class SqlTableRW {
       cols_.emplace_back(name, type, 255, nullable, oid);
   }
 
+  void AddColumn(transaction::TransactionContext *txn, std::string name, type::TypeId type, bool nullable,
```

Comment:
`AddColumn` does not support adding varlen columns. You should also test adding varlen columns to make sure your code is handling varlens correctly.